css-shapes-2/Overview.bs

```diff
@@ -165,6 +168,79 @@ Supported Shapes</h3>
 					in the path string.
 					For the initial direction follow SVG 1.1.
 			</ul>
+		<dt>
+			<pre class=prod>
+				<dfn>shape()</dfn> = shape( [<<fill-rule>>,]? from <<coordinate-pair>>, <<draw-command>>#)
```

Comment:
I'd drop the comma after fill-rule, it's cleaner without.

Reply:
Fine with me!